src/Domain/Structure/OrientationMap.hpp

```diff
@@ -70,6 +70,22 @@ class OrientationMap {
   /// The corresponding Mesh in the neighbor
   Mesh<VolumeDim> operator()(const Mesh<VolumeDim>& mesh) const;
 
+  /// The corresponding Mesh in the neighbor
```

Comment:
I assume you commented out the above operator to make sure you changed all of the calls to it to your new function

Reply:
yep. I deleted the new function, since I have verified that I caught all occurrences.